test/cases/api.sh

```diff
@@ -238,7 +238,7 @@ function createReqFileAWS() {
       "architecture": "$ARCH",
       "image_type": "ami",
       "repositories": $(jq ".\"$ARCH\"" /usr/share/tests/osbuild-composer/repositories/"$DISTRO".json),
-      "upload_requests": [
+      "upload_request": 
         {
```

Comment:
formatting nitpick: could you move the `{` one line up, and then make sure the closing bracket has the same indentation as the now-removed `]` bracket?